deploy/template.yaml

```diff
@@ -325,6 +325,18 @@ Resources:
               Value: !GetAtt ApiGwHttpEndpoint.ApiEndpoint
             - Name: REDIS_SESSION_URL
               Value: !GetAtt RedisCache.RedisEndpoint.Address
+            - Name: GTM_ID
+            Value:
+             'Fn::If':
+               - 'IsProduction'
+               - 'GTM-TT5HDKV'
+               - 'GTM-TK92W68'
+             - Name: ANALYTICS_DOMAIN
+               Value:
+                 'Fn::If':
+                   - 'IsProduction'
+                   - 'account.gov.uk'
+                   - '${Environment}.account.gov.uk'
```

Comment:
When we deploy we should jump into the task definition to check these values have been set correctly